src/components/blocks/InformationBlock/InformationBlock.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import InformationBlock from './InformationBlock';
+
+export default {
+  title: 'Blocks/InformationBlock',
+  component: InformationBlock,
+  argTypes: {},
+} as ComponentMeta<typeof InformationBlock>;
+
+const Template: ComponentStory<typeof InformationBlock> = () => {
+  const data = [
+    {
+      id: 123,
+      value: 'foo',
+      label: 'bar',
+    },
+  ];
+
+  return (
+    <InformationBlock
```

Comment:
Let's add more stories:
1. `variant=main`
2. `variant=regular`
3. `dataInRow=false` (add at least two entries to `data`)
4. `dataInRow=true`